articles/hosted-pages/login/lock-passwordless.md

```diff
@@ -0,0 +1,21 @@
+---
+description: How to use Lock Passwordless with the Hosted Login Page
+---
+# Hosted Login with Lock Passwordless
+
+Using Lock Passwordless in the Hosted Login Page is the easiest and most secure implementation of passwordless authentication that can be achieved. Additionally, the Hosted Login Page is the _only_ way to perform passwordless authentication with native applications at this time.
```

Comment:
I would also add a note linking to another doc, for background info on Passwordless